src/test/java/reposense/git/GitLogTest.java

```diff
@@ -64,17 +64,68 @@ public void gitLog_includeAllJavaFilesAuthorIgnoreMovedFile_success() {
         Author ignoreMovedFileAuthor = getAlphaAllAliasAuthor();
         ignoreMovedFileAuthor.setIgnoreGlobList(Collections.singletonList("**movedFile.java"));
 
-        String content = GitLog.get(config, ignoreMovedFileAuthor);
-        Assert.assertTrue(TestUtil.compareNumberExpectedCommitsToGitLogLines(6, content));
+        String content = GitLog.getWithFiles(config, ignoreMovedFileAuthor);
+        Assert.assertTrue(TestUtil.compareNumberFilesChanged(4, content));
     }
 
     @Test
     public void gitLog_authorIgnoreAllJavaFiles_success() {
+        config.setFormats(Collections.singletonList(new Format("java")));
         Author ignoreAllJavaFilesAuthor = getAlphaAllAliasAuthor();
         ignoreAllJavaFilesAuthor.setIgnoreGlobList(Collections.singletonList("*.java"));
 
-        String content = GitLog.get(config, ignoreAllJavaFilesAuthor);
-        Assert.assertTrue(TestUtil.compareNumberExpectedCommitsToGitLogLines(2, content));
+        String content = GitLog.getWithFiles(config, ignoreAllJavaFilesAuthor);
+        Assert.assertTrue(TestUtil.compareNumberFilesChanged(0, content));
+    }
+
+    @Test
+    public void gitLog_authorIgnoreValidFilePath_success() {
+        Author ignoreValidFileAuthor = getAlphaAllAliasAuthor();
+
+        // Existing file is ignored
+        ignoreValidFileAuthor.setIgnoreGlobList(Collections.singletonList("annotationTest.java"));
+        String content = GitLog.getWithFiles(config, ignoreValidFileAuthor);
+        Assert.assertTrue(TestUtil.compareNumberFilesChanged(6, content));
+
+        ignoreValidFileAuthor.setIgnoreGlobList(Collections.singletonList("**Test**"));
+        content = GitLog.getWithFiles(config, ignoreValidFileAuthor);
+        Assert.assertTrue(TestUtil.compareNumberFilesChanged(5, content));
+
+        // Files that do not exist
+        ignoreValidFileAuthor.setIgnoreGlobList(Collections.singletonList("**blameTest"));
+        content = GitLog.getWithFiles(config, ignoreValidFileAuthor);
+        Assert.assertTrue(TestUtil.compareNumberFilesChanged(7, content));
+
+        ignoreValidFileAuthor.setIgnoreGlobList(Collections.singletonList("nonExisting.txt"));
+        content = GitLog.getWithFiles(config, ignoreValidFileAuthor);
+        Assert.assertTrue(TestUtil.compareNumberFilesChanged(7, content));
+
+        ignoreValidFileAuthor.setIgnoreGlobList(Collections.singletonList("./java"));
```

Comment:
this can use any value file, prefix with the `./` to demonstrate that it is valid and working.
e.g. `./newPos` to show the clear difference between this and the invalid example